src/main/scala/me/rexim/morganey/ReplAutocompletion.scala

```diff
@@ -72,13 +73,18 @@ class ReplAutocompletion(globalContext: () => InterpreterContext) extends Comple
         .map { case (root, f) => (root, stripExtensionIfModuleFile(f)) }
         .filter { case (root, f) => fileNameFilter(f.getName) }
         .map(relativize)
+        .map(fixRelativePathForWindows)
         .map(relativeFileToLoadPath)
 
     def relativize(baseAndFile: (File, File)): String = {
       val (base, file) = baseAndFile
       base.toURI().relativize(file.toURI()).getPath()
     }
 
+    def fixRelativePathForWindows(path: String): String =
+      if (runningOnWindows) path.replace('/', File.separatorChar)
```

Comment:
Hm... Probably. I would consult with Mr @ForNeVeR regarding that. Since he's an ultimate Windows expert. But he's already sleeping, so he will answer tomorrow. :)

In the meantime I'm gonna merge this PR once CI is passed.